config/config.go

```diff
@@ -11,8 +11,11 @@ var cfg *Config
 // Config represents service configuration for dp-frontend-dataset-controller
 type Config struct {
 	BindAddr                   string        `envconfig:"BIND_ADDR"`
-	RendererURL                string        `envconfig:"RENDERER_URL"`
+	Debug                      bool          `envconfig:"DEBUG"`
 	APIRouterURL               string        `envconfig:"API_ROUTER_URL"`
+	SiteDomain                 string        `envconfig:"SITE_DOMAIN"`
+	PatternLibraryAssetsPath   string        `envconfig:"PATTERN_LIBRARY_ASSETS_PATH"`
+	SupportedLanguages         [2]string     `envconfig:"SUPPORTED_LANGUAGES"`
```

Comment:
Beware I think this might break if you attempt to parse a list of more than 2 entires. I'm not entirely sure what will happen but I guess best case scenario is values 3+ will be missing.

I think you should be able to omit the size value.

Reply:
I've got some small fixes to do and will add this to the list, I don't think it's worth applying a hotfix though